src/linter.ts

```diff
@@ -202,6 +202,24 @@ export default class ShellCheckProvider {
         delayer.trigger(() => this.runLint(textDocument));
     }
 
+    private getSystemSpecificDocumentFileName(documentFileName: string): string{
+        if (process.platform == "win32"){
```

Comment:
I assume it will break for non-linux subsystem users?

Reply:
Yes, obviously. It's probably a good idea to add a note in the readme for Windows users or make the WSL integration optional (because shellcheck can in fact be installed natively in Windows, but I never saw it done)

Reply:
@sylveon IMHO it's better to add a new option, I think you can have a look at https://github.com/Microsoft/vscode-node-debug2/pull/142/files